google_api_availability_platform_interface/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## next
+
+* Fixed publishing warning'>=2.15.0 <3.0.0' to '>=2.15.0 <4.0.0'.
```

Comment:
The version number changed in the last commit, but forgot to update the message in the changelog:

```suggestion
* Fixed publishing warning and updated Dart SDK dependency from '>=2.15.0 <3.0.0' to '^3.5.0'.
```